source/systems/happinessandsaturation.md

```diff
@@ -10,17 +10,21 @@ The citizens, however, will NOT die if their saturation goes down to 0, they wil
 
 ## Happiness System
 
-There is an **overall** colony happiness and an **individual** worker happiness.
+There is an **overall** colony happiness and an **individual** citizen happiness. Higher colony happiness increases the initial level and skills of new colonists. Higher citizen happiness increases the rate the colonist's gains Intelligence when studying at a [Library](../../source/buildings/library).
 
 Overall colony happiness is 1-10 (initially set to 5). Happiness depends on three basic factors: **security, housing, and saturation**.
 
-Every night the overall colony happiness is recalculated based on the following:
+If a colonist's saturation above 7 shanks, they will be more happy.
+If the colonist's home (or for [guards](../../source/workers/guard), or [trainees](../../source/workers/archerintraining), workplace) is above level 2, the colonist will be more happy.
```

Comment:
```suggestion
If a colonist's home (or for [Guards](../../source/workers/guard) or trainees, their workplace) is above level 3, the colonist will be happier.
```
Isn't this level 3? And since there are also Knights in Training, maybe remove the link entirely?

Reply:
The mechanical rule is BuildingLevel / 2.5, so they should return blue happiness at 2.5 and green happiness at level 2.75 or 3.  The distinction mostly matters for the Barracks, since it's the only building that can have a partial building level.